f2/src/components/formik/warningModal/index.js

```diff
@@ -0,0 +1,73 @@
+import React, { useState } from 'react'
+import { Prompt } from 'react-router-dom'
+import { Container, Row, Modal } from 'react-bootstrap'
+import { Route } from 'react-router-dom'
+import { Trans } from '@lingui/macro'
+import { DefaultButton } from '../button'
+
+export const WarningModal = (props) => {
+  const [showModal, setShowModal] = useState(false)
+  const [confirmNav, setConfirmNav] = useState(false)
+
+  const confirm = (history) => {
+    setShowModal(false)
+    setConfirmNav(true)
+    history.goBack()
+  }
+
+  const cancel = () => {
+    setShowModal(false)
+  }
+
+  const shouldPrompt = (dirty, isSubmitting) => {
+    const alteredForm = dirty && !isSubmitting
+
+    console.log(alteredForm && !confirmNav)
```

Comment:
please don't forget to remove the " console.log " line before merge if you want

Reply:
Removed.